packages/mui-joy/src/Autocomplete/Autocomplete.test.tsx

```diff
@@ -480,11 +480,14 @@ describe('Joy <Autocomplete />', () => {
           renderTags={(value, getTagProps) =>
             value
               .filter((x, index) => index === 1)
-              .map((option, index) => (
-                <Chip key={index} endDecorator={<ChipDelete {...getTagProps({ index })} />}>
-                  {option.title}
-                </Chip>
-              ))
+              .map((option, index) => {
+                const { key, ...tagProps } = getTagProps({ index });
+                return (
+                  <Chip key={key} endDecorator={<ChipDelete {...tagProps} />}>
+                    {option.title}
+                  </Chip>
+                );
+              })
```

Comment:
I think this test didn't fail in #42047. The `key` prop is already being passed separately.

Reply:
My bad, I'll reverse this change

Reply:
Reversed